portfolio/src/main/webapp/script-home.js

```diff
@@ -49,3 +49,14 @@ function main() {
           slideShowGallery.adjustSlideManual(ADJUST_FORWARD); 
     }
 }
+
+/* 
+ * A function for a button click, changes the button to header, Who Am I, 
+ * by gathering data from servlet.
+ */
+function fetchHeader() {
```

Comment:
Pick a different name than "fetchHeader". You dont want to carelessly use well defined terms. "header" is already an HTML name so this function sounds like its returning an HTML header but its literally just  modifying the contents of an element.

Consider: 
fetchIntro()

or something else more descriptive